tests/test_apps.py

```diff
@@ -43,6 +43,14 @@ def test_insert_single_word(app, input_str, expected):
     assert (node.prefix, node.isWord) == (expected, True)
 
 
+def test_search_empty_string(app):
+    # Searching with empty string should resturn empty list
+    term = ''
+    app.search(term)
+    app.update_top_results()
+    assert len(app.search(term)) == 0
```

Comment:
nit: I think we should start using `unittest` module

Reply:
yeah, we are currently using `pytest`